lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -116,9 +116,10 @@
                                 title: gettext('Time Zone'),
                                 valueAttribute: 'time_zone',
                                 helpMessage: gettext(
-                                    'Select the time zone for displaying course dates. If you do not specify a ' +
-                                    'time zone here, course dates, including assignment deadlines, are displayed in ' +
-                                    'Coordinated Universal Time (UTC).'
+                                    'Select the time zone for displaying course dates. ' +
```

Comment:
In addition to changing the help text, we also need to rename the text in the dropdown for timezones.  So the "blank" item now says something like "local setting".  @sstack22 Please confirm.